packages/components/src/theme/typography.ts

```diff
@@ -0,0 +1,73 @@
+import { PaletteMode } from '@mui/material';
+import { TypographyOptions } from '@mui/material/styles/createTypography';
+import { common } from './colors';
+
+export const typography = (mode: PaletteMode): TypographyOptions => {
+  return {
+    fontFamily: ['Qanelas Soft Regular', 'Roboto', 'Helvectica', 'Arial', 'sans-serif'].join(','),
+    body1: {
+      fontFamily: ['Open Sans', 'Roboto', 'Helvectica', 'Arial', 'sans-serif'].join(','),
+      fontSize: '16px'
```

Comment:
@Rexford74 do we have our fonts defined in `rem`? If so, could you update the list here?

Reply:
Yes they are defined in both rem and px. Not only font sizes though.. even line heights are defined in rem and px as well. The font weight for the different sizes was also specified, but I think it is still subject to review.

Reply:
`rem` and only `rem` is needed.

Reply:
Well, actually, including px was for designers using Sistent since rem is not a unit for designing in Figma. I included the rem units specifically because of the engineering end of things to make for seamless handoff.
If I misunderstand your point, please kindly clarify. However, this is the reasoning behind having both units available on Sistent.